examples/Python/snowboydecoder_arecord.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+
+import collections
+import pyaudio
```

Comment:
Since the whole point of using `arecord` is to get ride of `pyaudio`, why not remove it in one go?